python/packages/nisar/products/insar/InSAR_base_writer.py

```diff
@@ -361,12 +361,21 @@ def add_RSLC_to_procinfo_params_group(self, rslc_name: str):
                                   ' composite of data collected in multiple'
                                   ' radar modes, "False" otherwise')
         ds_params = [
+            DatasetParams(
+                "rfiMitigation",
+                np.bytes_(rfi_mitigation),
```

Comment:
```suggestion
                "rfiMitigation",
                rfi_mitigation,
```
There is no need of the np.bytes_ conversion here since the `add_dataset_and_attrs`  will convert the string to np.bytes_ type automatically

Reply:
Awesome. Thank you, @xhuang-jpl, for clarifying!